teddy.js

```diff
@@ -542,6 +558,29 @@
         teddy.renderedTemplates[template][l - 1].renderedTemplate = renderedTemplate
       }
 
+      // remove {! comments !} and (optionally) unnecessary whitespace
```

Comment:
Be sure to update the API documentation to reflect these changes via https://github.com/rooseveltframework/teddy#api-documentation